packager/common.gypi

```diff
@@ -122,6 +128,22 @@
           '-Werror',
         ],
       }],
+      ['static_link_binaries==1', {
+        'conditions': [
+          ['OS=="linux"', {
+            'defines': [
+              # Even when we are not using musl or uClibc, pretending to use
+              # uClibc on Linux is the only way to disable certain Chromium
+              # base features, such as hooking into malloc.  Hooking into
+              # malloc, in turn, fails when we are linking statically.
+              '__UCLIBC__',
+            ],
+          }],
+        ],
+        'ldflags': [
+          '-static',
```

Comment:
What is the binary size impact of the "fully" static build?

Reply:
Answering this question, I found out that we have not been stripping the release binaries.  So I'll be adding that to the workflow, as well.

Here are the sizes from Linux arm64 as an example:

| libpackager | build config | exe linking | stripped | size |
| ----- | ----- | ----- | ----- | ----- |
| static | release | dynamic | no | 6.7MB |
| static | release | dynamic | yes | 5.1MB |
| static | release | static | no | 8.5MB |
| static | release | static | yes | 5.8MB |

So the increase for stripped builds moving to static linking is 700kB, but since we didn't strip the executables before, the size will actually go down by 900kB once we do.

Reply:
Ah, thanks!